packages/redux-window-message-thunk/README.md

```diff
@@ -0,0 +1,67 @@
+# `redux-window-message-thunk`
+
+A middleware that allows you to setup event handlers for window 'message' events.
+
+## Philosophy
+We like thunks and want to use them wherever possible.  We should be able to handle all errors that occur inside of executing thunks, both sync and async in a centralized place.  When we handle an error, the error handling function is just an action creator that may itself return another thunk.
```

Comment:
This needs to be either updated to explain the window message thunks, or removed.

Reply:
Updated with a description specific to this package